proto/src/determined/resourcepool/v1/resourcepool.proto

```diff
@@ -98,6 +98,8 @@ message ResourcePool {
   ResourcePoolType type = 3;
   // The number of agents running in the resource pool
   int32 num_agents = 4;
+  // The number of slots in the resource pool
+  int32 num_slots = 33;
```

Comment:
isn't this the same as `slots_available`? it stands for the total number of slots for agent RM and used this way in the web UI unless I am mistaken.

it seems to be we should be just filling in that field instead of creating a new one.

Reply:
seems right to me, but i glanced the code and im unsure how `k.agent.numSlots()` is right in k8s. i think calling summarize may still be right, but yeah use `slots_available` not add `num_slots`?

Reply:
yep. on a related note, I may rip out `k.agent` out of here because it does not bring any value, and make `agentState` private to `internal/resourcemanagers/agent`. are you ok with that or do you think it'd be useful in some way for Slurm?

Reply:
very ok with it, i was not planning on having an equivalent for slurm.

Reply:
thanks for pointing this all out.